### PR TITLE
Updated architecture & implementation

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
Update the architectural design and implementation:
image

to give:
image

#### Ticket Link
None

### CODE REVIEW

File: src/components/IncidentDetails/IncidentDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            name = user.first_name + ' ' + user.last_name;
+          }
+
+          return (
+            <div
+              key={statusPost.id}
+              className={classes.statusUpdate}
+            >
+              <Text>
+                <div dangerouslySetInnerHTML={{ __html: writer.render(reader.parse(post.message)) }} />
```
Comment (jwilander) [inline src/components/IncidentDetails/IncidentDetails.tsx:120]: Does commonmark protect against XSS?

**Review by asaadmahmood — APPROVED** [review]
LGTM

**Review by jwilander — APPROVED** [review]
Great stuff! Just one comment

File: src/components/IncidentDetails/IncidentDetails.tsx (hunk 1 above)
Comment (lieut-data) [reply src/components/IncidentDetails/IncidentDetails.tsx:120]: Excellent catch! It does, but only if you explicitly ask for `safe: true` mode -- added.

**Pull request merged**